httptesting/http.go

```diff
@@ -81,6 +81,9 @@ type JSONCallParams struct {
 	// result.
 	ExpectBody interface{}
 
+	// ExpectHeader holds some expected HTTP headers in the response.
```

Comment:
// ExpectHeader holds any HTTP headers that must be present in the response.
// Note that the response may contain headers not in this field.